packages/web-components/src/components/va-modal/va-modal.scss

```diff
@@ -43,7 +107,7 @@
 
 .va-modal-close {
   background-color: transparent;
-  color: var(--color-primary);
+  color: var(--color-gray-dark);
   font-size: 2.25rem;
   padding: 0;
   position: absolute;
```

Comment:
Which close icon should we be using?

**USWDS Website**
![Screenshot 2023-04-13 at 3 02 34 PM](https://user-images.githubusercontent.com/872479/231870550-439066b2-37ab-4fce-a488-ff658f70cc51.png)

**VA Storybook**
![Screenshot 2023-04-13 at 3 02 28 PM](https://user-images.githubusercontent.com/872479/231870551-1591ee62-3597-46c4-bb5a-fdb1f36d7f76.png)

Reply:
For the mvp we are sticking with font awesome icons

Reply:
I remember @humancompanion-usds saying he wanted the FontAwesome icon and not the v3 icon. But I could be wrong. @humancompanion-usds which one you want us to use?